scene/gui/spin_box.h

```diff
@@ -52,11 +52,11 @@ class SpinBox : public Range {
 	void _line_edit_input(const Ref<InputEvent> &p_event);
 
 	struct Drag {
-		float base_val;
-		bool allowed;
-		bool enabled;
+		float base_val = 0;
+		bool allowed = false;
+		bool enabled = false;
```

Comment:
There's `enabled = true;` in the constructor which will override this.
I guess this should maybe be initialized to `true` and the constructor initialization removed, to keep the same behavior.

Reply:
I moved the fix to the constructor for consistency. Otherwise identical behavior to what I had before.

Reply:
Actually I must have misread, the constructor has `drag.enabled = false;`, not `true`.

It is indeed better to do it here than in the constructor, so I suggest reverting back to the previous version but removing the re-initialization in the constructor.

Reply:
Alright, I moved the initialization back to the struct.